### ISSUE TITLE
Suggested Edits for "Recording and Measuring Your Musical Progress with R"

### ISSUE THREAD

**Issue by chronoclast** [Emelieh21/r-tutorials#1]
# First Part

## Style & Typos
- Adding subsections may improve readability (i.e. _titles_ per every section).
- The link to Alison Sparrow's video plays the video straight at second 18; revise the link, that means that there's a field with `&t=18s` at the end.

## Suggested Edits
- 2nd paragraph: Remove the comma in the sentence _"One of the examples used, was the recording of a G-scale played by a violin."_
- Besides, "G scale" does not need a dash, only on the filename `G-scale.wav`.
- 3rd paragraph: _"...converting a recording to those specifications is very easy with the online-converter."_ I would rephrase this, and mention "encoding settings" instead of specifications.
- 4th paragraph: _"First of all, let's read the original G-scale.wav file into R."_ Replace for "with R", as "read into R" doesn't sound good.
- 4th paragraph: _"...at least on my PC this works - I have a 32 bit Windows 10 laptop."_; I would rephrase for something like "it worked great in my 32-bit Windows 10 system, and since the package has been recently released, it surely will work in most modern setups as well" (or something like that, to give more confidence).

**Comment by chronoclast** [comment]
# Second Part

## Style & Typos
- 1st code block: Remove the space between "be" and "ware" in _"Be ware that the window..."_.
- Turn "expNotes" into `expNotes`.
- 7th paragraph: Replace _"comparible"_ for "comparable".
- 7th paragraph: Add a space in _"soundfile"_ (i.e. it should be "sound file").
- 9th paragraph: Replace _"automize"_ for "automate".

## Suggested Edits
- 5th paragraph: I'd rephrase this _"Great! And with the tuneR package it is actually possible to play the sound straight from R."_ Maybe something like "note how the last line of this code snippet is actually playing the sound file straight from R".
- 5th paragraph: _"In the article from Vessy..."_; here, I would say something like "in the previously mentioned article, this is done with the function `transcribeMusic()`, which also creates a **melodyplot** of the recording".
- 5th paragraph: since it's a way of speaking, or your own expression, maybe add italics on "Singstar feeling".
- 5th paragraph: This is generally messed up: _"Note: it might an extra dependency is needed (the pastecs package) - but this is easily solved with `install.packages("pastecs")`."_
- 7th paragraph: _"Let's quickly inspect the originalSound file"_; I'd just say "let's have a look at it". Or at least, add code style on the file name `originalSound`, otherwise it's a bit confusing.
- 8th paragraph: Replace _"Now we know the duration..."_ for "Now that we know the duration...".
- 9th paragraph: In this part _"Since I want to keep track of my recordings over a longer period of time - I collect the results of each recording session in a csv."_ first I would replace the dash for a comma (no need for that), and then I'd say "csv file" instead of just "csv".

**Comment by chronoclast** [comment]
# Third Part

## Style & Typos
- 10th paragraph, code block: In the inline comment `...the red line show what I played...`, there's an "s" missing. The correct form is `...the red line shows what I played...`.

## Suggested Edits
- 11th paragraph: Replace _"Meaning - we can treat..."_ for "This means that we can treat...".
- Add a conclusion or "outro" at the end; it feels like the article ends in a rough way. Maybe something like: "As you can see, tracking your musical progress with R is not only possible, but also fun. Besides, it's a great learning experience, and a good chance to explore topics like _blah blah blah_...".

In short: Good stuff! Just some _makeup_ here and there, and it'll be ready to roll! 🎉
